### PR TITLE
[10.0][FIX]Update business_requirement_resource_template.py

### PR DESCRIPTION
**Pull request by joycehuan**
fix business_requirement_resource_template model  category_id link to project_category

### CODE REVIEW

**Comment by lonelysun** [comment]
@joycehuan  fix the PR name,  [FIX]***

**Comment by lonelysun** [comment]
@elicoidal 
For fix issue #281

File: business_requirement_deliverable_resource_template_categ/models/business_requirement_resource_template.py [hunk 1]
```diff
@@ -8,6 +8,6 @@ class BusinessRequirementResourceTemplate(models.Model):
     _inherit = "business.requirement.resource.template"
 
     categ_id = fields.Many2one(
-        'project.task',
+        'project.category',
```
Comment (elicoidal) [inline business_requirement_deliverable_resource_template_categ/models/business_requirement_resource_template.py:11]: LGTM

**Review by elicoidal — APPROVED** [review]
Bump the version number

**Comment by elicoidal** [comment]
@victormartinelicocorp @lonelysun

**Review by martintorresvm — CHANGES_REQUESTED** [review]
LGTM but
increase version number
add your name on the collaborators

**Comment by joycehuan** [comment]
@elicoidal @victormartinelicocorp I've fixed the PR name, [FIX]*** and added  name on the collaborators

**Review by lonelysun — APPROVED** [review]

**Comment by elicoidal** [comment]
@victormartinelicocorp your remarks are attended I think we can move forward

**Pull request merged**